Makefile

```diff
@@ -121,13 +136,20 @@ caasp4os-deploy:
 caasp-prepare:
 	scripts/caasp_prepare.sh
 
-.PHONY: all-caasp4os
-all-caasp4os: deps-caasp4os caasp4os-deploy caasp-prepare gen-config chart scf login
-
 .PHONY: clean-caasp4os
 clean-caasp4os:
 	scripts/caasp4os_destroy.sh
 
-.PHONY: eirini-release
-eirini-release:
-	scripts/eirini-release.sh
+# pipeline targets:
+
+.PHONY: kind
+kind: clean deps up kubeconfig
+
+.PHONY: all
+all: kind setup chart gen-config scf login
+
+.PHONY: dind
```

Comment:
That's not used by any pipeline (but is the one that must be used when running from the docker image)

Reply:
renamed the comment from "pipeline targets" to "full targets"